File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int l =0;
        for (int i =0 ; i<nums.length; i++){
            if (nums [l] != nums[i]){
                nums [l + 1] = nums[i];
                l++;
            }
        }
        return l +1;

    }
}

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        dic = {}
        counter = 0 
        for i in range (0,len(nums)):
            if nums[i] not in dic.keys():
                nums[counter] = nums[i]
                counter += 1
                dic[nums[i]] = 1
        return counter